README.md

```diff
@@ -1,48 +1,26 @@
 
-Like other “___ Wars” competitions, Hydro uses the concept of “gauges”- deployed liquidity is split between proposals proportional to the number of votes that they receive, instead of a winner-takes-all approach.
-
-Atom is commonly used to enter and exit positions of other Cosmos tokens. Supplying more liquidity will help to cement Atom’s role as interchain money. The competition in Hydro to secure these PoL spots will generate excitement around Atom, and incentivize holders to lock it up.
-
-Projects using the Cosmos Hub’s Interchain Security platform will receive a gauge multiplier. For a given percentage of the vote, they will receive a higher percentage of the deployed PoL.
-
-The advantage of PoL compared to issuing incentives (i.e. paying rewards to liquidity providers in certain pools) is that
-the community pool does not *spend* its funds, but rather deploys them. This means that the funds can be re-used in the future, and the community pool can grow over time.
 
 # Main features
 
 The main features of the Hydro smart contract are as follows:
 
 ## Locking
 
-Users can lock their Atoms in the Hydro smart contract to receive voting power. Concretely, users can lock liquid staking module shares aka LSM shares in Hydro. Each LSM share represents a staked Atom that is delegated to a certain validator. Learn more about the Liquid Staking Module [here](https://github.com/iqlusioninc/liquidity-staking-module).
-This means users that currently have liquid Atom tokens first need to stake their tokens before they can lock them in Hydro.
-There are different locking durations available, and the longer the duration, the more voting power the user receives.
-Currently, the different options for locking are 1, 2, 3, 6, and 12 months.
-Once the locking period is over, the user can unlock their locked tokens and will receive the original amount of tokens back.
-
-It is also possible to refresh a partially expired lockup to gain more voting power (read more about *Voting* below).
+Users can lock their Atoms in the Hydro smart contract to receive voting power. Concretely, users can lock liquid staking module shares aka LSM shares in Hydro. Each LSM share represents a staked Atom that is delegated to a certain validator. Learn more about the Liquid Staking Module [here](https://github.com/iqlusioninc/liquidity-staking-module). This means users that currently have liquid Atom tokens first need to stake their tokens before they can lock them in Hydro. There are different locking durations available, and the longer the duration, the more voting power the user receives. Once the locking period is over, the user can unlock their locked tokens and will receive the original amount of tokens back. It is also possible to refresh a partially expired lockup to gain more voting power (read more about *Voting* below).
```

Comment:
```suggestion
Users can lock their ATOMs in the Hydro smart contract to receive voting power. 
Concretely, users can lock Liquid Staking Module (LSM) shares in Hydro. 
Each LSM share represents a staked ATOM that is delegated to a certain validator. 
Learn more about the LSM [here](https://hub.cosmos.network/main/modules/lsm-staking). 
This means users that currently have liquid ATOM tokens first need to stake their tokens before they can lock them in Hydro. 
There are different locking durations available, and the longer the duration, the more voting power the user receives. 
Once the locking period is over, the user can unlock their locked tokens and will receive the original amount of tokens back. 
It is also possible to refresh a partially expired lockup to gain more voting power (read more about *Voting* below).
```
Do not use the iqlusion repo as it's probably outdated.

Reply:
linked forum post